backend/tracim_backend/lib/webdav/authentification.py

```diff
@@ -47,7 +47,7 @@ def isRealmUser(
         session = environ["tracim_dbsession"]  # type: Session
         api = UserApi(None, session, self.app_config)
         try:
-            api.get_one_by_email(username)
+            api.get(email=username, username=username)
```

Comment:
I know this is correct but this looks fishy. I'd say:

1. rename this variable to `username_or_email`, `webdavuser` or something.
2. test for the presence of `@` in this variable and use the specific function (whether email or username). This could probably be done in a method of `UserApi`.

Reply:
i do suggest rename this variable to `login` as login is the name used for this kind of thing.

Reply:
1. I'm not sure renaming the variable is possible as the `isRealmUser` method is called by external code which could use a keyword parameter call.
2. I started with your idea but then saw this `get()` method, so decided to use it instead of creating yet another method to get a user. Even if I'm not too fond of it… I'll have a look at its usages, if not too risky/time-consuming I'll remove it in favor of a `get_from_login()` method where login can be email or username.

Reply:
I removed the get() method as it wasn't used in many cases.

Reply:
Ok for me to remove this method indeed